text/0000-placement-by-return.md

```diff
@@ -732,6 +710,40 @@ impl<T> Drop for Filled<T> {
 }
 ```
 
+## Synergy with RFC-1909
+
+This proposal was written to be independent from
+[RFC-1909 (unsized rvalues)](https://github.com/rust-lang/rfcs/blob/master/text/1909-unsized-rvalues.md).
+
+While the two their
```

Comment:
Dangling sentence is probably not what you want.

Reply:
Oops.